index.html

```diff
@@ -34,236 +36,178 @@
-          I can help you build a product , feature or website Look through some
-          of my work and experience! If you like what you see and have a project
-          you need coded, don’t hesitate to contact me.
-        </p>
-        <ul class="display-flex-row-wrap justify-center font-23-px text-primary list-style-none flex-gap-5 ">
-            <li> 
-                <a href=https://twitter.com/iamasum369" class="text-primary"> 
-                    <i class="fa fa-twitter "></i>
-                </a> 
+      </div>
+    </nav>
+  </header>
+  <main>
+    <section id="hero-section" class="display-flex-col">
+      <h1 class="text-center">
+        <span class="text-primary"> Hey there. I’m AiAsum. </span> <br />
+        <span class="text-green"> I’m a software developer </span>
+      </h1>
+      <p id="hero-text" class="text-center padding-20-px font-20-px text-primary">
+        I can help you build a product , feature or website Look through some
+        of my work and experience! If you like what you see and have a project
+        you need coded, don’t hesitate to contact me.
+      </p>
+      <ul class="display-flex-row-wrap justify-center font-23-px text-primary list-style-none flex-gap-5 ">
+        <li>
+          <a href=https://twitter.com/iamasum369" class="text-primary">
+            <i class="fa fa-twitter "></i>
+          </a>
+        </li>
+        <li>
+          <a href=https://www.linkedin.com/in/victorasumdev/" class="text-primary">
+            <i class="fa fa-linkedin"></i>
+          </a>
         </li>
-        <li> 
-            <a href=https://www.linkedin.com/in/victorasumdev/" class="text-primary"> 
-                <i class="fa fa-linkedin"></i>
-            </a> 
-    </li>
 
-    <li> 
-        <a href=https://www.linkedin.com/in/victorasumdev/" class="text-primary"> 
+        <li>
+          <a href=https://www.linkedin.com/in/victorasumdev/" class="text-primary">
             <i class="fa fa-medium"></i>
-        </a> 
-</li>
-<li> 
-    <a href=https://www.linkedin.com/in/victorasumdev/" class="text-primary"> 
-        <i class="fa fa-github"></i>
-    </a> 
-</li>    
-          
-<li> 
-    <a href=https://www.linkedin.com/in/victorasumdev/" class="text-primary"> 
-        <i class="fa fa-angellist"></i>
-    </a> 
-</li>         
-           
-        </ul>
-        <button type='button' type="button" id="call-to-action" class="font-23-px margin-top-5 self-center">
-            <i class='fa fa-angle-down'></i>
-        </button>
-      </section>
-      <section id="work-section">
-        <h2 class="font-bold text-capitalize text-center">My Recent Works</h2>
-        <ul id="card-container" class="display-grid">
-           <li class="card">
-            <div class="card-content">
-              <h4 class="project-title text-primary text-center">Multi-Post Stories Gain+Glory</h4>
-              <ul class=" tag-language-container list-style-none ">
```

Comment:
Hi Victor, you need to convert the section of your entire work to incorporate dynamic javascript by utilizing the data in your javascript objects to display it on your html.Your works section still has a lot of HTML in it.
<img width="901" alt="Screen Shot 2023-05-25 at 9 15 18 AM" src="https://github.com/AsumVictor/Porfolio/assets/37291194/38f74d14-6588-47fb-9227-dd000ef81993">